drivers/hmis/app/graphql/types/hmis_schema/ce_match_rule.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeMatchRule < Types::BaseObject
+    # object is a Hmis::Ce::Match::Rule
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :type, HmisSchema::Enums::CeMatchRuleType, null: false, method: :rule_type
+    field :owner_type, String, null: false
+
+    def owner_type
```

Comment:
I believe my original intent was that a given rule might be owned by a polymorphic owner, where owner is the entity under which the rule is managed (coc, project). This would determine who could manage the rule and where it might appear in the UI. 

But a rule might apply based on multiple attributes related to the opportunity (project_types, funders, etc). This applicability is different from ownership.

So if the intention is to show the owner type, this resolver isn't quite right, it would instead return object.owner_type.

If the intention is to describe the types of opportunities this rule would target, this isn't quite right either, it wouldn't be project_type or funder but it would need to describe all the non-empty keys of applicability_config.

Reply:
I'm not really seeing the need for distinguishing between "ownership" and "applicability". This probably needs some more discussion and requirement-gathering. I would suggest leaving this as-is for now, and revisit when we get into the "requirements management" chunk of work.